docs/src/main/paradox/stream/operators/Source-or-Flow/onErrorResume.md

```diff
@@ -0,0 +1,32 @@
+# onErrorResume
+
+Allows transforming a failure signal into a stream of elements provided by a factory function.
```

Comment:
Should you mention that this is Java Only, that Scala users should use `recoverWith`?